src/gui/ogr/qgsvectorlayersaveasdialog.cpp

```diff
@@ -489,7 +497,7 @@ void QgsVectorLayerSaveAsDialog::mFormatComboBox_currentIndexChanged( int idx )
   else if ( leLayername->text().isEmpty() &&
             !mFilename->filePath().isEmpty() )
```

Comment:
If `mFilename->filePath().isEmpty()` we still want to use the laundered layer's name.